hardware.json

```diff
@@ -45,6 +114,7 @@
         }
     },
     "webserver" : {
+        "enabled": true,
         "lighttpd" : {
             "configFile": "/etc/lighttpd/lighttpd.conf",
             "wifiSetupConfig": "/etc/lighttpd/lighttpd-wifisetup.conf",
```

Comment:
Same as above.

Reply:
This configuration is required, otherwise the webserver cannot be started / stopped from the settings menu ;-)

Reply:
I know, I'm talking about this: `"wifiSetupConfig": "/etc/lighttpd/lighttpd-wifisetup.conf",`